docs/docs/code/nodejs/README.md

```diff
@@ -65,9 +63,11 @@ This communicates a couple of key concepts:
 If you're using [step parameters](/workflows/steps/#passing-data-to-steps-step-parameters) or [connect an account to a step](/connected-accounts/#from-a-code-step), you may notice two new parameters passed to the function signature, `params` and `auths`:
 
 ```javascript
-async (event, steps, params, auths) => {
-  // this node.js code will execute when your workflow is triggered
-};
+defineComponent({
+  async run({ steps, $ }) {
+    // this node.js code will execute when your workflow is triggered
+  })
+});
 ```
 
 When you use [step parameters](/workflows/steps/#passing-data-to-steps-step-parameters), Pipedream passes the `params` object (named pairs of param key and its associated value) to the function.
```

Comment:
Similar to the above, this also needs to be changed to reflect props instead of params, no function signature

Reply:
Params migrated to props 👍